src/layout/MainLayout/Drawer/DrawerFooter/index.tsx

```diff
@@ -6,24 +6,26 @@
  */
 
 // material-ui
-import { ListItemButton } from '@mui/material';
+// import { ListItemButton } from '@mui/material';
```

Comment:
Can we remove the lines instead of commenting them out?

Reply:
Yes, done